low-level/framework/utils/store_queue.py

```diff
@@ -104,7 +114,10 @@ def put(self, item):
             logger.debug("StoreQueue, put, consul memory usage exceded limit, \
                 removing old message")
             self._create_space(size_of_item)
-        store.put(item, f"SSPL_UNSENT_MESSAGES/{self.tail}", pickled=False)
+        # store.put(item, f"SSPL_UNSENT_MESSAGES/{self.tail}", pickled=False)
+        SSPL_UNSENT_MESSAGES = os.path.join(self.cache_dir_path, f"SSPL_UNSENT_MESSAGES_{self.tail}")
+        #store.put(item, SSPL_UNSENT_MESSAGES, pickled=False)
+        store.put(item, SSPL_UNSENT_MESSAGES)
```

Comment:
Same comment as store.get() above, need to understand how this would append to the tail of the queue? or file as using fileStore?